contracts/pool-manager/sim/lp_fee.py

```
@@ -0,0 +1,178 @@
+    print(f'new_balances: {new_balances}')
+    # Invariant after change
+    D1 = get_D(new_balances, amp)
+    assert D1 > D0
+
+    # We need to recalculate the invariant accounting for fees
+    # to calculate fair user's share
+    fees = []
+    mint_amount = 0
+
+    if total_supply > 0:
+        ideal_balance: uint256 = 0
```

Ruff (0.8.2)

Undefined name `uint256`

(F821)